src/iop/splittoning.c

```diff
@@ -97,6 +97,16 @@ int default_colorspace(dt_iop_module_t *self, dt_dev_pixelpipe_t *pipe, dt_dev_p
   return iop_cs_rgb;
 }
 
+const char *description(struct dt_iop_module_t *self)
+{
+  return dt_iop_set_description(self, _("create a two color linear toning effect where the shadows and highlights\n"
```

Comment:
"use two specific colors for shadows and highlights and create a linear toning effect between them up to a pivot."

Just a proposal.